docs/kubernetes/how-to-deploy-istio-with-kubernetes/index.md

```diff
@@ -67,27 +67,27 @@ Come hang out with us in #linode on the Kubernetes Slack! http://slack.k8s.io/
 
 1.  If you visit the [Linode Cloud Manager](https://cloud.linode.com/), you will see your newly created cluster nodes on the Linodes listing page.
 
-### Install Helm and Tiller
+### Install Helm
 
-Follow the instructions in the [How to Install Apps on Kubernetes with Helm](https://www.linode.com/docs/kubernetes/how-to-install-apps-on-kubernetes-with-helm/) guide to install Helm and Tiller on your cluster. Stop before the section on "Using Helm Charts to Install Apps".
+Follow the instructions in the [How to Install Apps on Kubernetes with Helm](https://www.linode.com/docs/kubernetes/how-to-install-apps-on-kubernetes-with-helm-3/) guide to install Helm on your cluster. Stop before the section on "Using Helm Charts to Install Apps".
```

Comment:
We should always remove the linode.com portion of the link, so that it stays relative. So, /docs/kubernetes/how-to-install-apps-on-kubernetes-with-helm-3/

Reply:
Updated this in 3 places

Reply:
updated this in 3 places